src/test/java/uk/gov/hmcts/reform/blobrouter/reconciliation/service/ReconciliationServiceTest.java

```diff
@@ -95,6 +100,28 @@ void should_throw_exception_when_supplier_statement_is_invalid() throws Exceptio
             .hasMessageContaining("Failed to process Supplier statement");
     }
 
+    @Test
+    void should_throw_exception_when_supplier_statement_has_invalid_containers() throws Exception {
+        // given
+        given(serviceConfig.getSourceContainers()).willReturn(List.of("bulkscan"));
+        var supplierStatement = new SupplierStatement(
+            List.of(
+                new Envelope("a.zip", null, "c1", "C1", singletonList("12"), singletonList("1")),
+                new Envelope("q.x", null, "BULKSCAN", "B", singletonList("4"), singletonList("3"))
```

Comment:
let's reference valid container names. here an in all other tests. (instead of relying on a string that happens to be valid)